include_core/omrhookable.h

```diff
@@ -65,12 +70,35 @@ typedef struct J9HookInterface {
 #define J9HOOK_AGENTID_DEFAULT  ((uintptr_t)1)
 #define J9HOOK_AGENTID_LAST  ((uintptr_t)-1)
 
+/* time threshold (=100 milliseconds) for triggering the tracepoint  */
+#define DEFAULT_THRESHOLD_IN_MILLISECONDS_WARNNING_CALLBACK_ELAPSED_TIME	100
```

Comment:
This should be namespaced as well - either 'OMR' or 'J9' (to match existing constants in the file)

ie: `OMRHOOK_DEFAULT_THRESHOLD_IN_MILLISECONDS_WARNNING_CALLBACK_ELAPSED_TIME`